apps/src/aiComponentLibrary/chatMessage/ChatMessage.tsx

```diff
@@ -28,6 +28,7 @@ const ChatMessage: React.FunctionComponent<ChatMessageProps> = ({
 
   const hasDangerStyle =
     status === Status.PROFANITY_VIOLATION ||
+    status === Status.USER_INPUT_TOO_LARGE ||
```

Comment:
Thought for later - might be good to move this logic out of the shared `ChatMessage` component and into aichat logic, and just pass in a string with the desired error message here. That way the different chat applications don't necessarily need to have the same set of logical statuses and this component can be more presentation focused.

Reply:
Sounds good - i'll create a jira ticket for this.

Reply:
https://codedotorg.atlassian.net/browse/LABS-997